docs/app/get-started/install-cypress.mdx

```diff
@@ -1,164 +1,85 @@
-You select the Node.js version using the Docker image tag.
+If you're using a [Cypress Docker image](../continuous-integration/overview#Cypress-Docker-variants), you'll find a fixed version of Node.js is pre-installed in the image. You select the Node.js version using the Docker image tag.
 
 ### Package Manager
 
 Cypress is [installed](#Install) using one of the following supported package managers:
 
-| Package Manager                                  | Version             | Installation instructions                                                                                       |
-| ------------------------------------------------ | ------------------- | --------------------------------------------------------------------------------------------------------------- |
-| [npm](https://docs.npmjs.com/)                   | `10.1.0` and above  | [Downloading and installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) |
-| [Yarn 1 (Classic)](https://classic.yarnpkg.com/) | `1.22.22` and above | [Yarn 1 (Classic) Installation](https://classic.yarnpkg.com/en/docs/install)                                    |
-| [Yarn (Modern aka berry)](https://yarnpkg.com/)  | `4.x` and above     | [Yarn Installation](https://yarnpkg.com/getting-started/install)                                                |
-| [pnpm](https://pnpm.io/)                         | `8.x` and above     | [pnpm Installation](https://pnpm.io/installation)                                                               |
+| Manager                                          | Version    | Docs                                                                             |
+| ------------------------------------------------ | ---------- | -------------------------------------------------------------------------------- |
+| [npm](https://docs.npmjs.com/)                   | `10.1.0`+  | [Install npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) |
+| [Yarn 1 (Classic)](https://classic.yarnpkg.com/) | `1.22.22`+ | [Install Yarn Classic](https://classic.yarnpkg.com/en/docs/install)              |
+| [Yarn (Modern aka berry)](https://yarnpkg.com/)  | `4.x`+     | [Install Yarn Modern](https://yarnpkg.com/getting-started/install)               |
+| [pnpm](https://pnpm.io/)                         | `8.x`+     | [Install pnpm](https://pnpm.io/installation)                                     |
 
-#### Yarn Configuration
+#### Yarn users
 
 [Yarn (Modern)](https://yarnpkg.com/) configuration using [nodeLinker: "node-modules"](https://yarnpkg.com/configuration/yarnrc#nodeLinker)
 is preferred. Cypress [Component Testing](/app/core-concepts/testing-types#What-is-Component-Testing) is not currently compatible with the default setting [nodeLinker: "pnp"](https://yarnpkg.com/configuration/yarnrc#nodeLinker) which uses [Yarn Plug'n'Play](https://yarnpkg.com/features/pnp).
 
-#### pnpm Configuration
+#### pnpm users
```

Comment:
Renaming this section heading according to the audience it addresses, instead of describing the contents, causes a broken anchor:

```text
  - Broken anchor on source page path = /app/references/error-messages:
     -> linking to /app/get-started/install-cypress#pnpm-Configuration
```

Consider reverting to the original section heading text. If not, then the `error-messages` page needs changing.